test/library/packages/csv/CSV.chpl

```diff
@@ -57,8 +57,8 @@ module CSV {
         if line.size == 0 then
           continue;
         const vals = line.split(sep);
-        for param i in 1..numFields(t) {
-          getFieldRef(r, i) = vals[i]: getField(r, i).type;
```

Comment:
This suggests that maybe we should adjust `getFieldRef` to be zero based as well in the future?

Reply:
Boy, it sure does.  I think I started to take that on at some point during this whole process, then worried I was trying to do too much at once and backed away from it.  I may try to squeak that in this weekend...

Reply:
(I think I also put this off because I ran into it when looking at the __primitive() forms of these, forgetting that they had a user-facing form as well).

Reply:
I submitted https://github.com/chapel-lang/chapel/issues/15427 to capture this item.